app/main.py

```diff
@@ -1,2 +1,70 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity: int = 8
+        self.size: int = 0
+        self.load_factor: float = 0.66
+        self.hash_table: list = [None] * self.capacity
+
+    def __len__(self) -> int:
+        print(self.capacity)
+        return self.size
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        threshhold = int(self.capacity * self.load_factor)
+        if self.capacity - threshhold > self.hash_table.count(None):
+            self.resize()
+        if not isinstance(key, Hashable):
+            raise TypeError(f"unhashable type: '{type(key)}'")
+        index = self.get_index_from_hash(key)
+        while True:
+            if not self.hash_table[index]:
+                self.hash_table[index] = (key, hash(key), value)
+                self.size += 1
+                break
+            elif self.hash_table[index]:
+                if self.hash_table[index][0] == key:
+                    self.hash_table[index] = (key, hash(key), value)
+                    break
+                if index + 1 < self.capacity:
+                    index += 1
+                else:
+                    index = 0
+
+    def __getitem__(self, key: Hashable) -> Any:
+        if not isinstance(key, Hashable):
+            raise TypeError(f"unhashable type: '{type(key)}'")
```

Comment:
No need to make this check and raise an error. It will raise the same error in the next line of code, because in get_index_from_hash method you will try to get a hash from this value

Reply:
Thank you, I thought I have to raise it by myself. I'll fix it!